tests/cypress/fixtures/groovy/prepareSchedulerTest.groovy

```diff
@@ -0,0 +1,29 @@
+import org.jahia.services.scheduler.SchedulerService
+import org.jahia.registries.ServicesRegistry
+import org.jahia.services.scheduler.BackgroundJob
+import org.quartz.JobDetail
+import org.quartz.JobExecutionContext
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+Map<String, Object> jobData = new HashMap<>();
+jobData.put("foo", "bar");
+JobDetail job = BackgroundJob.createJahiaJob("Test job", TestJob.class);
+job.setName("JOB_NAME") // pass job name as variable
+job.getJobDataMap().putAll(jobData);
+
+// schedule
+SchedulerService schedulerService = ServicesRegistry.getInstance().getSchedulerService();
+schedulerService.scheduleJobNow(job);
+
+class TestJob extends BackgroundJob {
```

Comment:
This class is not available from within the `ModulesResourceLoaderClassLoadHelper`, causing the default `NoOpJob` class to be used instead.
How can we load a custom `BackgroundJob` in our Cypress tests?